tests/filename_map.yml

```diff
@@ -0,0 +1,96 @@
+salt/modules/(apk|aptpkg|ebuild|dpkg|freebsdpkg|mac_brew|mac_ports|openbsdpkg|opkg|pacman|pkgin|pkgng|pkg_resource|rpm|solarisips|solarispkg|win_pkg|xbpspkg|yumpkg|zypper)\.py:
+  - unit.states.test_pkg
+  - integration.modules.test_pkg
+  - integration.states.test_pkg
+  - integration.states.test_pkgrepo
+
+salt/modules/(mac_user|useradd|pw_user|solaris_user|win_useradd)\.py:
+  - unit.states.test_user
+  - integration.states.test_user
```

Comment:
Yes, this is all on purpose. runtests.py will automagically match files that match a naming convention. Otherwise, the map file would be thousands of lines long and would be a pain in the butt to maintain.

So, there is no need to add `integration.modules.test_useradd` and `unit.modules.test_useradd` as they would be picked up when `salt/modules/useradd.py` is specified in the `--from-filenames` parameter.